src/devtools/client/debugger/src/components/Editor/SourcemapToggle.tsx

```diff
@@ -47,15 +47,10 @@ export function SourcemapToggle({ selectedSource, setModal, showAlternateSource
   );
 }
 
-const connector = connect(
-  (state: UIState) => ({
-    selectedSource: getSelectedSourceWithContent(state)!,
-  }),
-  {
-    showAlternateSource: actions.showAlternateSource,
-    setModal,
-  }
-);
+const connector = connect(() => ({}), {
```

Comment:
:hand: Can just do `null` for `mapState` here

Reply:
Thanks! It's been so long since I've written a connect statement 😆 I couldn't remember the correct way to omit this.